ios/Video/RCTVideo.swift

```diff
@@ -68,6 +68,7 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     private var _presentingViewController: UIViewController?
     private var _pictureInPictureEnabled = false
     private var _startPosition: Float64 = -1
+    private var _playerOutput: AVPlayerItemVideoOutput?
```

Comment:
Does it increase memory usage significantly ?

Reply:
As I know, it depends on resolution or bitrate